### DISCUSSION TITLE
Import settings from Windows PuTTY?

### DISCUSSION THREAD

**Discussion by TKruzze** [jlesage/docker-putty#14] [Q&A] [answered]
Are there any tools, tips, or tricks to import my settings from my Windows version of PuTTY? I've got a ton of them and would rather not have to manually create.

**Comment by jlesage** [comment, marked as answer]
Sessions are saved into the registry under Windows, while under Linux they are saved as individual files.

I found the following (old) site that shows how to convert PuTTY sessions from Windows to Linux: https://mdinh.wordpress.com/2013/06/07/migrating-windows-putty-registry-to-linux/

If you manage to get Linux sessions files, you can copy them under `/config/.putty/sessions/` inside the container.

**Reply by TKruzze** [reply]
Thanks, I'll give it a shot! 

EDIT: Method works fine.
